litestar/openapi/plugins.py

```diff
@@ -0,0 +1,552 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+import msgspec
+import yaml
+
+from litestar.enums import MediaType, OpenAPIMediaType
+from litestar.handlers import get
+from litestar.serialization import encode_json, get_serializer
+
+if TYPE_CHECKING:
+    from litestar.connection import Request
+    from litestar.router import Router
+
+__all__ = (
+    "JsonRenderPlugin",
+    "OpenAPIRenderPlugin",
+    "RapidocRenderPlugin",
+    "RedocRenderPlugin",
+    "StoplightRenderPlugin",
+    "SwaggerRenderPlugin",
+    "YamlRenderPlugin",
+)
+
+
+class OpenAPIRenderPlugin:
+    def __init__(
+        self,
+        *,
+        path: str | tuple[str, ...],
```

Comment:
Why `tuple[str, ...]`?

Reply:
it was typed as `str | list[str]` which is the same typing as our handler paths, however providing a default value of `=["openapi.yaml", "openapi.yml"]` would be flagged by linters as providing a mutable default value.

I'll type it `Sequence[str]` instead.